Assets/!/Prefabs/Level/Scripts/Map Loading Script.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class MapLoadingScript : MonoBehaviour
+{
+    [SerializeField] GameObject[] mapSegments;
+    private GameObject[] ambientList;
```

Comment:
Let's stick with C# conventions:
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names

```suggestion
    private GameObject[] _ambientList;
```

Reply:
Is this correct now?

Reply:
I think so xD